api/desecapi/pdns.py

```diff
@@ -81,9 +82,11 @@ def get_keys(domain):
     Retrieves a dict representation of the DNSSEC key information
     """
     r = _pdns_get(NSLORD, '/zones/%s/cryptokeys' % pdns_id(domain.name))
-    return [{k: key[k] for k in ('dnskey', 'ds', 'flags', 'keytype')}
+    content = [{k: key[k] for k in ('dnskey', 'ds', 'flags', 'keytype')}
             for key in r.json()
             if key['active'] and key['keytype'] in ['csk', 'ksk']]
+    metrics.get('desecapi_dnssec_key_information_dict_created').inc()
```

Comment:
This is redundant with `desecapi_pdns_keys_fetched`. Let's use the name `desecapi_pdns_keys_fetched`, but only here.

Also, we don't want to count the dictionary creation, but this specific `_pdns_get` call. So you can just add metric after the `_pdns_get` call, and leave the `return` statement unchanged.